staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -98,6 +98,11 @@ type Event struct {
 	// +optional
 	ImpersonatedUser *authnv1.UserInfo
 	// Source IPs, from where the request originated and intermediate proxies.
+	// The source IPs are listed from (in order):
+	// 1. X-Forwarded-For request header IPs
+	// 2. X-Real-Ip header, if not present in the X-Forwarded-For list
+	// 3. The remote address for the connection, if it doesn't match the last IP in the list
```

Comment:
this is confusing to read. You want to say "in the list up to here, i.e. 1 and 2".

Reply:
looks like this comment is still outstanding